tests/test_services.py

```diff
@@ -156,12 +153,9 @@ def test_remove_all_services_from_user(app_no_node: SpecterFlask, empty_data_fol
         config={},
     )
 
-    # The ServiceEncryptedStorageManager singleton creates a mess in the test suite. Have
-    # to override its (potentially) already-set values with references to this test's
-    # new instances and temp storage dir.
-    storage_manager = ServiceEncryptedStorageManager.get_instance()
-    storage_manager.data_folder = user_manager.data_folder
-    storage_manager.user_manager = user_manager
+    storage_manager = ServiceEncryptedStorageManager(
+        user_manager.data_folder, user_manager
+    )
```

Comment:
Could you add here instantiating and using a `ServiceUnencryptedStorageManager` too? I don't know it that works well, because they use the same json file.